src/NuGet.Core/NuGet.Protocol/Model/PackageSearchMetadataBuilder.cs

```diff
@@ -19,7 +19,6 @@ public class PackageSearchMetadataBuilder
         private readonly IPackageSearchMetadata _metadata;
         private AsyncLazy<IEnumerable<VersionInfo>> _lazyVersionsFactory;
         private AsyncLazy<PackageDeprecationMetadata> _lazyDeprecationFactory;
-
```

Comment:
similar to another type, you can revert these changes completely. 

`git checkout dev -- src/NuGet.Core/NuGet.Protocol/Model/PackageSearchMetadataBuilder.cs` should do it.